src/app/utilities/api-clients/teams.js

```diff
@@ -14,7 +14,19 @@ export default class teams {
     }
 
     static add(teamName) {
-        return http.post(`/zebedee/teams/${teamName}`).then(response => {
+        return http.post(`/zebedee/teams/${teamName}`, body).then(response => {
+            return response;
+        });
+    }
+
+    static createTeam(body) {
+        return http.post(`/groups`, body).then(response => {
+            return response;
+        });
+    }
+
+    static addMemberToTeam(teamID, userID) {
```

Comment:
this should be consistent you use  `addMemberToTeam` but the api is groups...

Reply:
Hi, yer so we are using AWS Cognito 'groups' for storing 'teams' and 'roles' (the new backend for Auth rather than Zebedee)

A user can be in a Cognito 'group' for a role e.g. admin, publisher (some others will exist) which grant access to different areas of Florence and content.
A user can also be in a group for a team (which when attached to a collection, gives the team access to content).

So where in Zebedee we had Roles and Teams as completely different things we now just have 'groups' and are using that for both roles and teams. (as you attach policies to groups that grant levels of access).

Content Designer was also consulted in regards to terminology around showing the user 'teams' or 'groups' and it was decided by them that team was more appropriate.

So we are calling off to add members to a team, but underneath the hood, teams are stored as 'groups'. I thought a good place to make that distinction in the code is when dealing with the API (so in the API client). So to Florence, everything is 'team' or 'role' like what is presented to users on the frontend, but the backed / APIs it is 'groups'.